homeassistant/components/media_player/squeezebox.py

```diff
@@ -173,7 +173,7 @@ def update(self):
     def volume_level(self):
         """ Volume level of the media player (0..1). """
         if 'mixer volume' in self._status:
-            return int(self._status['mixer volume']) / 100.0
+            return int(float(self._status['mixer volume'])) / 100.0
```

Comment:
We don't have to convert it to an interview anymore  if we're converting it to a float